### DISCUSSION TITLE
How to calculate the tax for pricing structures that have minimum user licenses.

### DISCUSSION THREAD

**Discussion by doransmestad** [stopthessotax/sso-wall-of-shame#13] [Ideas]
Hey @derekcwhite @daallgeier -  question on how we want to calculate the tax.  Inspired by Otter.ai here: https://github.com/stopthessotax/sso-wall-of-shame/pull/9. 

To properly handle companies that have a tiered per-user pricing structure, or like Otter.ai that may have equal pricing between "Business" and "Enterprise",  I propose that we evaluate the tax percentage based on a 5 (or 10) user count in carrying with the spirit of:

>  How is base pricing determined?
> We disregard free tier pricing, as we can assume these aren't intended for long term business customer use. We also disregard "single person" pricing, under the assumption that we're looking on behalf of a team of 5, 10, or more people. 

So if a company charged $10/user for the base, and then $10/user for the SSO tier, but mandated a minimum of 100 licenses, the tax would be:   ($10 * 100 users) / ($10 * 5 users)  =  2000% 

This does have the downside of not properly doing the tax percentage for larger user-count purchases, but I think I'm good with that trade off to push our goal of keeping the SSO charge under 10% (if it must exist at all).

Thoughts?

**Comment by daallgeier** [comment]
I would go and assume a 5 user minimum in such cases. You really care about (SAML) SSO for less users then that?

**Reply by doransmestad** [reply]
Nope, I'm good with 5 users as our base for calculations.  Sorry if I made it seem otherwise above there.  My primary goal is to agree on a way to call companies out for "same price" licenses but minimum user counts being the SSO tax.   Sounds like we're in agreement!
